### PR TITLE
Document Media Scale

### PR DESCRIPTION
**Pull request by MattiSG**

### CODE REVIEW

File: _pages/encyclopedia/3-qualification/3-tools.md [hunk 1]
```diff
@@ -13,6 +13,31 @@ permalink: /encyclopedia/qualification/tools
...
+
+The French Ambassador for Digital Affairs’ team offers a way to compensate that tendency: [Media Scale](https://github.com/ambanum/media-scale). Rather than focusing on the quantitative aspect, it offers a scale of relevance by putting the number of shares in perspective with reference articles shares for the cultural context.
+
+![Media Scale chatbot works by typing in a region code and a share count](/assets/img/3-qualification/3-tools/media-scale-chatbot-help.png)
+![For example, share fr 1004](/assets/img/3-qualification/3-tools/media-scale-chatbot-request.png)
+![1004 shares for an article in France is equivalent to “Le bassin de la Villette temporairement fermé à la baignade” by Le Monde in June 2017](/assets/img/3-qualification/3-tools/media-scale-chatbot-result.png)
+
+This more qualitative approach helps give perspective to readers of threat notices that might be further away from the topics, and prioritise your own qualification work.
```
Comment (BerylBogui) [inline _pages/encyclopedia/3-qualification/3-tools.md]: Replace your by their

**Comment by BerylBogui** [comment]
@MattiSG It's all good for me, ready to merge :)

File: _pages/encyclopedia/3-qualification/3-tools.md (hunk 1 above)
Comment (MattiSG) [reply _pages/encyclopedia/3-qualification/3-tools.md]: “your” might indeed not be very appropriate, but the aim was to distinguish the recipients of an alert (who are not very into the topic) and the teams doing the qualification work.

**Comment by MattiSG** [comment]
Awesome, thanks a lot for the improvements! 🤗

**Pull request merged**